packages/gatsby-theme-store/src/sdk/img/fileManager.ts

```diff
@@ -10,11 +10,17 @@ interface Options {
   quality?: number
 }
 
-export const optimize = (src: string, opts: Options) => {
+export const optimize = (src: string, opts: Options, ratio = 1) => {
   const url = new URL(src)
 
   for (const option of Object.keys(opts)) {
-    url.searchParams.append(option, `${opts[option as keyof Options]}`)
+    let paramValue = opts[option as keyof Options]
+
+    if (paramValue && ['width', 'height'].includes(option)) {
```

Comment:
```
['width', 'height'].includes(option)
```
creates an array and makes a find in each interaction. A simple comparison statement should be much faster
```suggestion
    if (option === 'width' || option === 'height') {
```

Reply:
Oh, good to know. I have this habit of using includes for better readability but didn't know about that.